R/antigen.garnish_predict.R

```diff
@@ -268,7 +268,8 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
 
     blastdt <- blastdt[highest == iedb_score] %>%
               .[!(nmer %like% "-")] %>%
```

Comment:
can we remove or fix `like "-"` instead of dropping silently?

Reply:
Blast puts out gapped alignments, and prior to this I sub out gaps with str_replace("-", "").  So this actually isnt necessary but is a holdover from an older iteration.  We can remove, I just kept cause im paranoid.

Reply:
fixed